docs/user/stream.rst

```diff
@@ -21,11 +21,23 @@ Modules
 -  ``geomesa-stream-datastore`` - ``DataStore`` implementation
 -  ``geomesa-stream-gs-plugin`` - GeoServer hooks for stream sources
 
-Usage
------
+Using GeoMesa Stream
+--------------------
+
+Installing into GeoServer
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
the underline here doesn't match the length of the header, so rst is going to complain.